service/experiment-requests.yaml

```diff
@@ -18,6 +18,14 @@
 # You can run "make presubmit" to do basic validation on this file.
 # Please add new experiment requests towards the top of this file.
 
+- experiment: 2020-10-16
+  fuzzers:
+    - aflplusplus
+    - aflplusplus_optimal
+    - aflplusplus_memcache_2000mb
+    - aflplusplus_memcache_2mb
+    - aflplusplus_memcache_pre
+
```

Comment:
Could you please add:
```
description: "<YOUR DESCRIPTION HERE>"
```
I want to test our new feature.

Reply:
done, and presubmit is happy